cmd/wallet-web/src/components/CredentialDetails/DeleteCredentialModal.vue

```diff
@@ -4,92 +4,58 @@
-                />
-              </g>
-            </svg>
-          </div>
-          <span class="pt-5 pb-3 text-lg font-bold text-neutrals-dark">
-            {{ t('CredentialDetails.deleteCredential') }}?
-          </span>
-          <div class="relative flex-auto">
-            <p class="pb-12 text-base text-center text-neutrals-medium">
-              {{ t('CredentialDetails.deleteCredentialConfirmMessage') }}
-            </p>
-          </div>
+  <modal :show="showModal">
+    <template #content>
+      <!--content-->
+      <div class="flex relative flex-col items-center px-8 pt-10 w-full">
+        <div class="flex justify-center items-center w-15 h-15 bg-primary-valencia rounded-full">
+          <svg width="32" height="32" xmlns="http://www.w3.org/2000/svg">
+            <g transform="translate(3 6)" fill="none" fill-rule="evenodd">
+              <rect stroke="#ffffff" stroke-width="2" x="1" y="1" width="24" height="19" rx="4" />
+              <ellipse fill="#ffffff" cx="8" cy="13" rx="4" ry="2" />
+              <circle fill="#ffffff" cx="8" cy="8" r="2" />
+              <path
+                stroke="#ffffff"
+                stroke-width="2"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                d="M16 8h5M16 13h2"
+              />
+            </g>
+          </svg>
         </div>
-        <!-- Buttons Container -->
-        <div
-          class="
-            md:flex-row
-            gap-4
-            justify-start
-            md:justify-between
-            items-center
-            px-5
-            md:px-8
-            pt-4
-            pb-5
-            text-center
-            bg-neutrals-magnolia
-            rounded-b-2xl
-            flex flex-col
-            modal-footer
-            border-t border-0 border-neutrals-lilacSoft
-          "
-        >
-          <button class="w-full md:w-auto lg:w-auto btn-outline" type="button" @click="closeModal">
-            {{ t('CredentialDetails.deleteButtonCancel') }}
-          </button>
-          <button
-            id="deleteButton"
-            class="order-first md:order-last lg:order-last w-full md:w-auto lg:w-auto btn-danger"
-            type="button"
-            @click="deleteCredential(credentialId)"
-          >
-            {{ t('CredentialDetails.deleteButtonLabel') }}
-          </button>
+        <span class="pt-5 pb-3 text-lg font-bold text-neutrals-dark">
+          {{ t('CredentialDetails.deleteCredential') }}?
+        </span>
+        <div class="relative flex-auto">
+          <p class="pb-12 text-base text-center text-neutrals-medium">
+            {{ t('CredentialDetails.deleteCredentialConfirmMessage') }}
+          </p>
         </div>
       </div>
-    </div>
+    </template>
+    <!-- Buttons Container -->
+    <template #actionButton>
+      <styled-button
+        id="deleteButton"
+        class="order-first md:order-last lg:order-last w-full md:w-auto lg:w-auto"
```

Comment:
You have added excessive classes for breakpoints above `md` again. Please, remember that `md` already applies to everything above this brake point, so you do not need to specify the same class for `lg`